distributed/node.py

```diff
@@ -87,6 +87,21 @@ def _setup_logging(self, logger):
         logger.addHandler(self._deque_handler)
         weakref.finalize(self, logger.removeHandler, self._deque_handler)
 
+        loggers_to_register = dask.config.get("distributed.admin.log-events")
+
+        for logger_name, level in loggers_to_register.items():
```

Comment:
This is currently very awkward but the loggers passed in this methods are only `distributed.scheduler` and `distributed.worker`. Most interesting exception logs actually pop up in `distributed.core` and `tornado.application`.

The _proper_ way to do this would be to attach the EventHandler to the logger `distributed` and ensure that all loggers we're using are propagating their msgs https://docs.python.org/3.8/library/logging.html#logging.Logger.propagate This is the default of the stdlib but it is user configuration *and* we are overwriting it for some reason for "old style" logging config https://github.com/dask/distributed/blob/5c3eacd768545d84e865340f08c6965f521541df/distributed/config.py#L101

Without this knowledge, the only sane way I see is to list the registered loggers explicitly. The same goes for the deque handler, btw, but I didn't touch it, yet.